publish_docker_images.sh

```diff
@@ -0,0 +1,41 @@
+\#!/bin/bash
+publish_bionic () {
+    date_string=`date +"%Y%m%d"`
+    bionic_sha=$(docker images | grep "drake-torch         bionic" | awk '{print $3}')
```

Comment:
The spacing here between `drake-torch` and `bionic` is different in my terminal. Maybe there's a better way to check if these two are on the same line while being agnostic about spacing?